lib/core/src/Cardano/Wallet/Primitive/Types/UTxO.hs

```diff
@@ -166,6 +171,11 @@ isSubsetOf :: UTxO -> UTxO -> Bool
 isSubsetOf (UTxO a) (UTxO b) =
     a `Map.isSubmapOf` b
 
+-- | a ⊂ b
+isProperSubsetOf :: UTxO -> UTxO -> Bool
+isProperSubsetOf (UTxO a) (UTxO b) =
+    a `Map.isProperSubmapOf` b
```

Comment:
```suggestion
isProperSubsetOf (UTxO a) (UTxO b) = a `Map.isProperSubmapOf` b
```

Reply:
I've converted this to use avoid infix application in f36e0a45fea20f3e64b09ba26cdf76b8a66d4fd2.